.env

```diff
@@ -0,0 +1,7 @@
+    REACT_APP_DUMMYGRAM_APIKEY: "AIzaSyCMmQUVG213FeNgkZ3HSPAz1qahmZVy4Xw",
```

Comment:
Why is this file included? Is there any reason?

Reply:
I created this file soo that .env variables can be set for the project but got some error while doing this.
As you can see in #102 .
This template can be used in near future if some is able to access process.env variables